homeworks/HW_05/task3_2_for.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const constNumber = 39;
+let multiplyFunction = '';
+
+for (let count = 1; count <= 10; count++) {
+    multiplyFunction = constNumber * count;
+    console.log(`${constNumber} x ${count} = ${multiplyFunction}`);
```

Comment:
Можна і так, але найлегше було б просто не виводити результат множення до окремої змінної
```suggestion
    console.log(`${constNumber} x ${count} = ${constNumber * count}`);
```